config/helm/appmesh-controller/templates/deployment.yaml

```diff
@@ -79,6 +79,8 @@ spec:
         - --sds-uds-path={{ .Values.sds.udsPath }}
         - --enable-backend-groups={{ .Values.enableBackendGroups }}
         - --cluster-name={{ .Values.clusterName}}
+        - --use-aws-dual-stack-endpoint={{ .Values.useAwsDualStackEndpoint}}
+        - --use-aws-fips-endpoint={{ .Values.useAwsFIPSEndpoint}}
```

Comment:
We already have flags for use dual stack endpoint and use fips endpoint. Do we really need to add new flags for this?

Reply:
The flags which we have already are being used only for sidecar. These are used for only AWS services (AppMesh,EKS,CloudMap). I think sidecar and AWS services are independent to each other. so I think we need to add new flags.

Reply:
I really wonder if a customer is every going to want to set these two sets of flags to different values. If a customer wants to use FIPS I'd expect they'd want to use it both in the sidecar and in the controller. It might be worth gathering some input on this, but it would be a real shame if we added two sets of flags that the customer always ends up changing together to get the desired result.

Reply:
For FIPS, its straightforward to say we have just the one flag, but for dual stack its trickier. This is because we overlooked support for using dual stack endpoints in the controller, so we now have a breaking change where we are calling a new endpoint than the one we were calling before.

If we are ok with a slight breaking change for the dual stack endpoints (one might be able to argue this is really a bug fix since we are now doing what we say we are doing) then I think we should go ahead and use just the 1 set of flags.

Reply:
But if we are not, then having the 2 sets of flags is fine

Reply:
I think I'd take the bugfix vs. break side of that argument, I really think we need to hold the line on making App Mesh harder to configure if at all possible.

Reply:
since other flags are already released, so we got to proceed with these flags for AWS api's.